.github/workflows/linux.yml

```diff
@@ -1447,6 +1447,7 @@ jobs:
         env:
           TYPE: ${{ github.event_name == 'schedule' && 'nightly' || 'precommit'}}
           TEST_DEVICE: CPU
+          USE_SYSTEM_CACHE: False
```

Comment:
maybe we need to set path to model cache instead ? In this case we can store models on Azure shared drivers as well

Reply:
Can we utilize these shared drives now?

Reply:
What about cleaning this cache? In your suggestion we wouldn't clean it, right, but will we have enough memory on machine?